issue_title_desc.md

```
@@ -0,0 +1,58 @@
+# GitHub Issue & Pull Request Details
+
+This document contains the copy-paste ready content for both the Issue Report and the Pull Request, following the project's strict contribution guidelines.
+
+---
+
+## 1. GitHub Issue Details
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)